libflux/flux-core/src/semantic/tests/vectorize.rs

```diff
@@ -1,24 +1,33 @@
 use super::*;
 use crate::semantic::{
-    import::Packages,
+    bootstrap,
     nodes::{FunctionExpr, Package},
     walk::{walk, Node},
     AnalyzerConfig, Feature,
 };
+use std::path::PathBuf;
 
 fn analyzer_config() -> AnalyzerConfig {
     AnalyzerConfig {
         features: vec![
             Feature::VectorizedMap,
             Feature::VectorizedConditionals,
             Feature::VectorizedEqualityOps,
+            Feature::VectorizedFloat,
         ],
         ..AnalyzerConfig::default()
     }
 }
 
 fn vectorize(src: &str) -> anyhow::Result<Package> {
-    let mut analyzer = Analyzer::new(Default::default(), Packages::default(), analyzer_config());
+    let stdlib_path = PathBuf::from(concat!(env!("CARGO_MANIFEST_DIR"), "/../../stdlib"))
+        .canonicalize()
+        .expect("stdlib path");
+
+    let cfg = analyzer_config();
+    let (prelude, imports, _sem_pkgs) = bootstrap::infer_stdlib_dir(&stdlib_path, cfg.clone())?;
+
+    let mut analyzer = Analyzer::new((&prelude).into(), &imports, cfg);
```

Comment:
@Marwes This is the change I made allowing my tests to include calls to `float`. If I understood you during standup, your recommendation instead is to effectively hand-write the builtin definition for the bits each test needs.

I'm not sure how that should be done. Any advice?

Reply:
Huh, okay. Rewriting the flux source like the following, without the prelude configured does seem to work okay.

```
builtin float : (v: A) => float

(r) => ({ r with a: float(v: r._value) })
```

I'll make a change for this and make a note on #4997

Reply:
Yeah, there is also `imp` and `env` to let us set things like this up https://github.com/influxdata/flux/issues/4997#issuecomment-1216424229

Reply:
Moved this up to where we initialize the analyzer in 889816d1d4d952864b6eb7034a3f10761ee9f660 -- feels more official to do it this way :+1: